cmake/maybe_build_using_clangw.cmake

```diff
@@ -40,7 +40,7 @@ function(maybe_build_using_clangw OE_TARGET)
         -Wall -Werror -Wpointer-arith -Wconversion -Wextra -Wno-missing-field-initializers
         -fno-strict-aliasing
         -mxsave
-        -fno-builtin-malloc -fno-builtin-calloc)
+        -fno-builtin-malloc -fno-builtin-calloc -fno-builtin)
```

Comment:
@johnkord Do you think this flags needs to be in pkgconfig as well or no?

Reply:
Hmm, that's a good thing to consider. I'm not sure! It seems that would probably make sense, but I don't want to limit a user's ability to use compiler inline optimizations.  I think I'll ask Mike what he thinks.  @mikbras have any opinions on this topic? :)

Reply:
I would not force flags on the end user. If they want these flags, they can add them as needed. There are many reasons the end user might want to use built-ins in their own code.